packages/esm-patient-orders-app/src/hooks/useOrderPrice.test.ts

```diff
@@ -0,0 +1,40 @@
+import { act, renderHook } from '@testing-library/react';
+import { type FetchResponse, openmrsFetch } from '@openmrs/esm-framework';
+import { type OrderPriceData } from '../types/order';
+import { useOrderPrice } from './useOrderPrice';
+import { mockOrderPriceData } from '../../../../__mocks__/order-price-data.mock';
+
+const mockedOpenmrsFetch = jest.mocked(openmrsFetch);
+
+describe('useOrderPrice', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
```

Comment:
All mocks [automatically get cleared](https://github.com/openmrs/openmrs-esm-patient-chart/blob/main/jest.config.js#L6), so need to do it explicitly.